index.html

```diff
@@ -47,66 +45,23 @@
 
 
     <!------------ YOUR SCRIPTS GO HERE. ------------>
-
-    <!-- Malware Teaching Level Scripts and Scenes -->
-    <!-- <script src="/scenes/teachinglevelscript.js"></script> -->
-    
-    <!-- Kanaan's Malware Testing Level Script -->
-    <script defer src="scripts/hangman.js"></script>
-
-    <!-- Gallal's module scripts -->
     <script defer src="scripts/encryption_script.js"></script>
 </head>
 <body>
     <!-- Whole game world (made in Phaser3) -->
     <div id="gameContainer"></div>
 
     <!-- Example of an outside "game module" that is manipulated through a separate file -->
-    <!-- <div class="gameModule" id="module1" style="background-color:lightgreen"></div> -->
+    <div class="gameModule" id="module1" style="background-color:lightgreen"></div>
     <!-- And so forth... -->
 
 
     <!------------ YOUR DIVS GO HERE. ----------------->
+    <!-- Use these by id or override them altogether -->
+    <div class="gameModule" id="instruction0">
 
-    <!-- Kanaan's instructions module -->
```

Comment:
Your branch was behind the master branch when pull requesting:

* I merged Kanaan's and your other pull request into master.
* You modified your fork that was out of date (before successful mergings)
* Now that you pull request from an old fork: Kanaan's module gets delited